express-get/server.ts

```diff
@@ -0,0 +1,52 @@
+import express from 'express';
+import pg from 'pg';
+import { ClientError, errorMiddleware } from './lib/index.js';
+
+const db = new pg.Pool({
+  connectionString: 'postgres://dev:dev@localhost/pagila',
+  ssl: {
+    // Allow non-SSL traffic to localhost
+    rejectUnauthorized: false,
+  },
+});
+
+const app = express();
+
+app.get('/api/films', async (req, res, next) => {
+  try {
+    const sql = `
+    select * from "films"
+    order by "replacementCost" desc
+    limit 2
+    `;
+    const result = await db.query(sql);
+    const films = result.rows;
+    res.json(films);
+  } catch (err) {
+    next(err);
+  }
+});
+
+app.get('/api/films/:filmId', async (req, res, next) => {
+  try {
+    const { filmId } = req.params;
+    if (!Number.isInteger(+filmId)) {
+      throw new ClientError(400, 'filmId is not an integer');
+    }
+
+    const sql = `
+    select *  from "films" where "filmId" = $1
+    `;
+    const result = await db.query(sql);
+    const films = result.rows[0];
```

Comment:
You need to make sure you actually found a film to return. If not, send a 404 response.